src/install_all.sh

```diff
@@ -92,6 +93,19 @@ $Build_gsi && {
  ${CP} regional_gsi.fd/exec/test_nc_unlimdims.x        ../exec/regional_test_nc_unlimdims.x
```

Comment:
The regional_gsi.fd/exec path does not exist when I build this code. Please ensure that this script will run for a fresh checkout of the code.

Reply:
Comment out all the copies from regional_gsi.fd/exec.

This is from EMC GSI. Not what we are using now. But leave those items for future merging with EMC build.